client/src/components/JurisdictionAdmin/JurisdictionAdminView.tsx

```diff
@@ -29,6 +39,13 @@ const JurisdictionAdminView: React.FC = () => {
     jurisdictionId: string
   }>()
 
+  const auth = useAuthDataContext()
+  assert(auth?.user?.type === 'jurisdiction_admin')
+  const jurisdiction = auth.user.jurisdictions.find(
+    j => j.id === jurisdictionId
+  )
+  const { organizationId } = jurisdiction?.election || {}
```

Comment:
What do you think of packaging all this feature flagging logic in some kind of `useJurisdictionAdminFeatureFlag` hook that calls `useAuthDataContext` and extracts `organizationId` internally? E.g.

```ts
const isBatchInventoryEnabled = useIsJurisdictionAdminFeatureFlagEnabled(
  'BATCH_INVENTORY',
  jurisdictionId,
)
```

(Of course, there are lots of permutations to the above)

Reply:
Sure, I made a more specific hook to just this feature flag for now since it's the only one we have: cf37105b